src/game.cpp

```diff
@@ -7953,37 +7951,91 @@ static void butcher_submenu( const std::vector<map_stack::iterator> &corpses, in
                                    ? string_format( _( "Your best tool has %d fine cutting." ), factorD )
                                    :  _( "You have no fine cutting tool." );
 
+    bool has_skin = false;
+    bool has_organs = false;
+
+    if( corpse != -1 ) {
+        const mtype *dead_mon = corpses[corpse]->get_mtype();
+        if( dead_mon ) {
+            for( const harvest_entry &entry : dead_mon->harvest.obj() ) {
+                if( entry.drop == "skin" ) {
+                    has_skin = true;
+                }
+                if( entry.drop == "offal" ) {
+                    has_organs = true;
+                }
+            }
+        }
+    }
+
     uilist smenu;
     smenu.desc_enabled = true;
     smenu.text = _( "Choose type of butchery:" );
 
-    smenu.addentry_col( BUTCHER, enough_light, 'B', _( "Quick butchery" ), cut_time( BUTCHER ),
+    const std::string cannot_see = colorize( _( "can't see!" ), c_red );
```

Comment:
Maybe change it to something like "too dark"? Otherwise players might think that there are some problems with their eyes.

Reply:
Huh, so what happens if the player is boomered etc and actually cannot see? Will they be able to start butchering or is this message displayed?

Reply:
i didn't check with boomered, but i checked with a blindfold.